tc/sgx/trusted_worker_manager/enclave/signup_enclave_wpe.cpp

```diff
@@ -0,0 +1,298 @@
+/* Copyright 2020 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "enclave_t.h"
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include <sgx_tseal.h>
+#include <sgx_utils.h>
+#include <sgx_quote.h>
+
+#include "crypto.h"
+#include "error.h"
+#include "avalon_sgx_error.h"
+#include "tcf_error.h"
+#include "zero.h"
+#include "jsonvalue.h"
+#include "parson.h"
+
+#include "enclave_data.h"
+#include "enclave_utils.h"
+#include "signup_enclave_util.h"
+#include "verify-report.h"
+
+#define WPE_HASH_SIZE 32
```

Comment:
Use SGX_HASH_SIZE (defined in sgx_report.h) instead of WPE_HASH_SIZE

Reply:
Done